src/Microsoft.DotNet.Interactive/Commands/SendEditableCode.cs

```diff
@@ -5,13 +5,14 @@ namespace Microsoft.DotNet.Interactive.Commands;
 
 public class SendEditableCode : KernelCommand
 {
-    public string KernelName { get; }
-    public string Code { get;  }
-
-    public SendEditableCode(string kernelName, string code, string targetKernelName = "vscode")
+    public SendEditableCode(string kernelName, string code, string targetKernelName = null)
```

Comment:
Because of this, will you run the last cell in `NotebookTestScript.dib` to make sure we didn't break this feature?

Reply:
I didn't make a significant change here, just moved the constructor to the top of the class declaration.